### ISSUE TITLE
[Suggestion] Being able to use the search bar with an emoji

### ISSUE THREAD

**Issue by Prometheos2** [rugk/awesome-emoji-picker#81]
Could it be possible to "reverse-search" an emoji?

Let me explain, let's say we have a certain emoji (e.g. 😅), and we don't know where it is in the part, nor how would it be named.
So it might be a good idea (although very niche) to have the possibility to paste the emoji in the search bar, and the app shows us the emoji, so we can have it in the "frequently used" section, or even know if it's a supported emoji.

For example, the dizzy emoji (😵) is more something I would call "shocked", thus I'm not finding it with the search feature.
Another example would be the woozy emoji (🥴) and the pleading one (🥺), which are not on the app, are usually not known by their names, could be found more easily with such a feature, or at the very least, we would know they're not available.

_P.S.: About the woozy and pleading emojis, since I'm unsure whether they are actually on the app or not, I won't open an issue for their addition unless I'm confirmed that they are not._

**Comment by rugk** [comment]
Actually that is already possible somewhat, at least in the search bar. If you have enabled the emojipedia search, you can paste 😅 and then just press <kbd>enter</kbd> and it searches on emojipedia and then finds the emoji.

However, in the popup/suggestions list it does not appear, because the emoji library we use does not support it, actually. I've actually already requested support for this in https://github.com/missive/emoji-mart/issues/311.

Thus, this is blocked by https://github.com/missive/emoji-mart/issues/311

**Comment by Prometheos2** [comment]
I was unaware of that feature 😅, my bad

As for the two (supposedly) missing emojis, I suppose it is also on emoji-mart's part to fix?

**Comment by rugk** [comment]
Yes, see https://github.com/rugk/awesome-emoji-picker/issues/34.
